mergify_engine/actions/merge_base.py

```diff
@@ -54,13 +54,27 @@ class PriorityAliases(enum.Enum):
     high = 3000
 
 
+class StrictMergeParameter(enum.Enum):
+    true = True
+    false = False
+    smart = "smart"
```

Comment:
if you re-read the code you see `smart` is actually `smart+ordered` so you should remove it